setup.py

```diff
@@ -41,9 +41,9 @@ def run(self):
     long_description=readme(),
     author='LightStep',
     license='',
-    install_requires=['opentracing>=1.2.2', 'grpcio>=1.1.3', 'six>=1.10'],
+    install_requires=['opentracing>=1.2.2', 'grpcio>=1.8.1', 'six>=1.10'],
```

Comment:
Interceptors exists within `grpc` exactly since 1.8.1?

Reply:
I checked it's 1.8.0. And changed the requirement.